src/main/java/frc/robot/Robot.java

```diff
@@ -25,12 +25,12 @@ public class Robot extends TimedRobot {
   private final CANSparkMax m_leftMotor = new CANSparkMax(1, MotorType.kBrushless);
   private final CANSparkMax m_rightMotor = new CANSparkMax(2, MotorType.kBrushless);
 
-  //m_leftMotor.setIdleMode(IdleMode.kCoast);
+  m_leftMotor.setIdleMode(IdleMode.kCoast);
```

Comment:
Adding the setIdleMode call here will result in a syntax error. You can't call this method here, you have to call it inside of another method, in this case the init method would be the best place to do that.